omnibus/Gemfile.lock

```diff
@@ -1,6 +1,6 @@
 GIT
   remote: https://github.com/chef/omnibus-software.git
-  revision: af33863799451fa78788a7dfc7b17ce0686ec585
+  revision: d6accc79f166cc6fc552e07088951bf467c30989
```

Comment:
This omnibus-software SHA needs to be updated once the following PRs get merged.

https://github.com/chef/omnibus-software/pull/905
https://github.com/chef/omnibus-software/pull/906